target/scripts/helper-functions.sh

```diff
@@ -288,27 +288,65 @@ export -f _monitored_files_checksums
 
 # ? --------------------------------------------- General
 
+# Outputs the DNS label count (delimited by `.`) for the given input string.
+# Useful for determining an FQDN like `mail.example.com` (3), vs `example.com` (2).
+function _get_label_length
```

Comment:
I think, count is more suitable:

```suggestion
function _get_label_count
```

As `mail` (third level domain), `example` (second level domain) and `com` (top level) are all "domains", `domain_count` could also be considered:

```suggestion
function _get_domain_count
```

Reply:
> I think, count is more suitable:

Absolutely :+1: 

> domain_count could also be considered:

I find `domain` as a bit overloaded and easy for others to misinterpret :sweat_smile: 

FQDN does describe itself with domains like you mention, but domain names themselves and hostname, also as DNS labels. I prefer to go with DNS labels (as per RFC on DNS) for this. I feel that `label` is more clear about the count/separation than `domain` is.